src/tooling/piral-cli/src/apps/new-piral.ts

```diff
@@ -90,17 +90,17 @@ export interface NewPiralOptions {
 
 export const newPiralDefaults: NewPiralOptions = {
   app: './src/index.html',
-  registry: defaultRegistry,
+  registry: config.registry,
   framework: 'piral',
   target: '.',
   version: cliVersion,
   forceOverwrite: ForceOverwrite.no,
-  language: SourceLanguage.ts,
+  language: config.language,
   install: true,
   template: 'default',
   logLevel: LogLevels.info,
-  npmClient: undefined,
-  bundlerName: 'none',
+  npmClient: config.npmClient,
+  bundlerName: config.bundler,
```

Comment:
This is a problem. By default, `config.bundler` is `webpack`, but it should be `none` here (as we don't know / don't select any).

I'd say let's keep the `none` here. It will automatically do the right thing (this is only a pre-selection - the `config` field is used in the bundler selection anyway).

Reply:
OK.